projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -402,28 +402,31 @@ const VideoPressEdit = CoreVideoEdit =>
 				</Fragment>
 			);
 
-			if ( isUploading ) {
-				return (
-					<Fragment>
-						{ blockSettings }
-						<Loading text={ __( 'Uploading…', 'jetpack' ) } />
-					</Fragment>
-				);
-			}
+			const isFetching = ! isUploading && ( isFetchingMedia || isFetchingPreview );
+			const useCoreVideo = ! isUploading && ! isFetching && ( fallback || ! preview );
+			const useCoreVideoOrLoading = isUploading || isFetching || useCoreVideo;
```

Comment:
Maybe instead of `useCoreVideoOrLoading` this can be called `renderCoreVideoAndLoadingBlocks` because in these circumstances it isn't necessarily being used or visible.

There might be a better naming suggestion for that though, I don't necessarily love what I came up with 😄

Reply:
✅ I think renderCoreVideoAndLoadingBlocks is all right as it describes exactly what it's doing, I also updated "useCoreVideo" to "displayCoreVideoBlock" and "isFetching" to "isFetchingVideo"